fjord/l10n/utils.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+
+_COLLAPSE_RE = re.compile(r'\s+', re.UNICODE)
+
+
+def collapse_whitespace(message):
+    """Collapses consecutive whitespace into a single space"""
+    return _COLLAPSE_RE.sub(' ', message).strip()
```

Comment:
For single line the `re` version is slightly faster: 1.02 usec per loop vs 0.768 (for re)
For 10 lines the `splitlines` is significantly faster: 3.95 usec per loop vs 11.5 (for re)